src/NUnitFramework/tests/Internal/TestExecutionContextTests.cs

```diff
@@ -137,7 +137,25 @@ public void Cleanup()
         }
 #endif
 
-        #endregion
+#if !PORTABLE && !NETSTANDARD1_6
```

Comment:
If I understand correctly, #2010 is bringing System.Threading into the .NET Standard build as a "side effect." That will lead to a vast number of other changes, including this one. 😄